src/components/profile/index.tsx

```diff
@@ -0,0 +1,143 @@
+'use client'
+
+import { ReactNode, useCallback, useState } from 'react'
+import { Button } from '../ui/button'
+import { Dialog, DialogContent, DialogTrigger } from '../ui/dialog'
+import { cn } from '@/lib/utils'
+import { NavItemsId, profileNav } from './config'
+
+interface ProfileProps {
+  trigger: ReactNode
+}
+
+export default function Profile({ trigger }: ProfileProps) {
+  const [selectedNavId, setSelectedNavId] = useState<NavItemsId>('my-account')
+
+  const ProfileContent = useCallback(() => {
+    switch (selectedNavId) {
+      case 'my-account':
+        return <MyAccount />
+      case 'notification-config':
+        return <NotificationConfig />
+      default:
+        return null
+    }
+  }, [selectedNavId])
+
+  return (
+    <Dialog>
+      <DialogTrigger>{trigger}</DialogTrigger>
+      <DialogContent className="flex h-[560px] w-[800px] max-w-full overflow-hidden rounded-[12px] p-0 shadow-modal">
+        <div className="flex h-full w-[200px] flex-col justify-between bg-gray-01 px-[12px] pb-[30px] pt-[32px]">
+          <div className="mb-[27px] flex items-center gap-[8px] px-[10px]">
+            <LogoIcon />
+            <div className="flex flex-col">
+              <span className="text-[14px]">
+                <span className="font-bold">픽토스</span> 님
+              </span>
+              <span className="text-[10px] font-[500] text-gray-07">picktoss@gmail.com</span>
+            </div>
+          </div>
+
+          <div>
+            {profileNav.map((nav) => {
+              const { id, name, items, styles } = nav
+
+              return (
+                <div key={id} className={cn('border-b border-gray-04', styles)}>
```

Comment:
개인적인 스타일이긴 한데, 다음과 같이 구조분해로 인자 받으면 더 깔끔해지지 않나요?

```suggestion
            {profileNav.map(({ id, name, items, styles }) => (
                <div key={id} className={cn('border-b border-gray-04', styles)}>
```

Reply:
굳굳 적용했습니다!